bve/src/parse/mesh/errors.rs

```diff
@@ -1,16 +1,40 @@
+use crate::parse::Span;
+
+/// A single error in the parsing or evaluation of a mesh.
 #[derive(Debug, Clone, PartialEq)]
 pub struct MeshError {
+    /// Info about the exact error.
     pub kind: MeshErrorKind,
+    /// Location of the error within the file.
     pub span: Span,
 }
 
+/// Enum representing various types of errors encountered when parsing meshes
 #[derive(Debug, Clone, PartialEq)]
 pub enum MeshErrorKind {
-    UTF8 { column: Option<u64> },
+    /// Invalid UTF-8. May be Shift-JIS or other encoding.
+    UTF8 {
+        /// Column of the error. Only Optional due to the CSV library.
+        column: Option<u64>,
+    },
+    /// Index provided to vertex specific command is out of bounds.
```

Comment:
Index provided to vertex-specific command is out of bounds

Reply:
Note to self: change is the hyphen